controls/anssi.yml

```diff
@@ -162,34 +162,52 @@ controls:
     level: intermediary
     title: Partitioning type
     rules:
+    # this covers nodev options
+    - mount_option_nodev_nonroot_local_partitions
     # The recommended partitioning type is as follows:
     # / <without option> Root partition, contains the rest of the tree
     # /boot nosuid, nodev, noexec (optional noauto) Contains the kernel and the bootloader. No access required once the boot finished (except update)
+    - partition_for_boot
+    - mount_option_boot_nosuid
+    - mount_option_boot_noexec
+    - mount_option_boot_noauto
+
     # /opt nosuid, nodev (optional ro) Additional packages to the system.  Read-only editing if not used
+    - partition_for_opt
+    - mount_option_opt_nosuid
+
     # /tmp nosuid, nodev, noexec Temporary files. Must contain only non-executable elements. Cleaned after reboot
     - partition_for_tmp
     - mount_option_tmp_nosuid
-    - mount_option_tmp_nodev
     - mount_option_tmp_noexec
 
     # /srv nosuid, nodev (noexec, optional ro) Contains files served by a service type web, ftp, etc
+    - partition_for_srv
+    - mount_option_srv_nosuid
+
     # /home nosuid, nodev, noexec Contains the HOME users.  Read-only editing if not in use
     - partition_for_home
     - mount_option_home_nosuid
-    - mount_option_home_nodev
+    - mount_option_home_noexec
 
     # /proc hidepid = 1 Contains process information and the system
+
```

Comment:
What about this? Will there be a rule for it?

Reply:
Nice catch, there should be a rule for it, and actually, it should `hidepid=2`